android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ApplicationConfiguration.kt

```diff
@@ -32,6 +32,7 @@ data class ApplicationConfiguration(
   val syncInterval: Long = 15,
   val syncStrategies: List<String> = listOf(),
   val loginConfig: LoginConfig = LoginConfig(),
+  val listSyncConfig: ListSyncConfig = ListSyncConfig(),
```

Comment:
For new sync configurations we also need to add documentation for the docusite here https://docs.opensrp.io/engineering/android-app/configuring

Reply:
Oh yes sure @ndegwamartin I will do so

Reply:
Yes sure, I set the page number to 1 in the app_config.json